cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -504,8 +504,8 @@
 
        ;; Float comparisons/tests: cmp (b w l q) (reg addr imm) reg.
        (XmmCmpRmR (op SseOpcode)
-                  (src XmmMemAligned)
-                  (dst Xmm))
+                  (src1 Xmm)
```

Comment:
Can we add to the comment above a note about arg order here, e.g. "compares with `src1` as LHS and `src2` as RHS, so that e.g. a less-than comparison tests `src1 < src2`."